lyrics/pop/holy water.txt

```diff
@@ -0,0 +1,15 @@
+Oh, my mother, holy water
+Wash away your sins, I sip away my sins
+And she said
+"Oh, my daughter, oh, I love you"
+Sip away your sins
+But I do what I want so I guess I'm gonna die young
+
+When I get home will you leave the lights on?
```

Comment:
How about changing "lights on" to leave the door open? @igachungi

Reply:
Sure!